packages/EasyCodingStandard/packages/SniffRunner/src/File/File.php

```diff
@@ -120,6 +120,11 @@ public function getErrors(): void
      */
     public function addFixableError($error, $stackPtr, $code, $data = [], $severity = 0): bool
     {
+        $fullyQualifiedCode = $this->currentSniffProvider->getSniffClass() . '.' . $code;
+        if ($this->skipper->shouldSkipCodeAndFile($fullyQualifiedCode, $this->path)) {
+            return false;
+        }
+
         $this->addError($error, $stackPtr, $code, $data, $severity, true);
```

Comment:
Maybe instead of your fix there should be

```php
return $this->addError($error, $stackPtr, $code, $data, $severity, true) && $this->isFixer;
```

What do you think?

Reply:
It felt bit weird to create duplicated code. This could be nicer.
Could you change it here? Just browser edit will do